packages/web-components/src/components/accordion/accordion.scss

```
@@ -23,20 +24,32 @@ $css--plex: true !default;
 :host(#{$prefix}-accordion-item),
 :host(#{$prefix}-accordion-item-skeleton) {
   @extend .#{$prefix}--accordion__item;
+  @include type-style('body-01');
 
   display: block;
   outline: none;
   .#{$prefix}--accordion__content {
     @extend .#{$prefix}--accordion__content;
 
     ::slotted(p) {
-      @include type-style('body-01');
-
       margin: 0;
+
+      font-size: var(
+        --cds-body-01-font-size
+      ) !important; /* stylelint-disable-line declaration-no-important */
+
+      font-weight: var(--cds-body-01-font-weight);
+      letter-spacing: var(--cds-body-01-letter-spacing);
+      line-height: var(--cds-body-01-line-height);
     }
   }
 }
 
+// Indicates this is the last visible item in a group (for styling the bottom border)
+:host([data-last-item]) {
+  border-block-end: 1px solid var(--cds-border-subtle);
```

Can these use the sass variables instead of the custom properties if the theme module is imported?

```diff
+ @use '@carbon/styles/scss/theme' as *;
```

```suggestion
  border-block-end: 1px solid $border-subtle;
```

---

Thank you!